### ISSUE TITLE
Workshop support would be appreciated.

### ISSUE THREAD

**Issue by thydungeonman** [samsface/godot-steam-api#40]
I didn't anticipate that some of my players would ask for it so I didn't mind that that this addon didn't support it. I don't really want to yank this addon out and put in a different one after the game has already launched so I'm in a bit of a rough spot.

**Comment by samsface** [comment]
Yeah a few people have asked now. I'll start working on it. Will update with an ETA when I'm a bit into to the work and can size it properly.

**Comment by thydungeonman** [comment]
Probably the bare minimum for uploading, subscribing, and unsubscribing/deleting workshop items would be good. Functions that query for download or upload progress are probably unnecessary right now as well as functions that track playtime and functions that search for workshop content (if I'm understanding those functions right they are just used for searching and subscribing to mods from inside the game which I'm probably not going to implement any time soon). 

Anyway I really appreciate you doing this! You can add ROGUE-FP to the list of games that use your addon if you want.
